sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -531,14 +534,15 @@ public async Task CreateAndGetAzureDataExplorerDataFeedWithAuthentication(AzureD
         }
 
         [RecordedTest]
-        [TestCase(AzureDataExplorerDataFeedSource.AuthenticationType.ServicePrincipal)]
-        [TestCase(AzureDataExplorerDataFeedSource.AuthenticationType.ServicePrincipalInKeyVault)]
-        public async Task CreateAndGetAzureDataExplorerDataFeedWithCredentialAuthentication(AzureDataExplorerDataFeedSource.AuthenticationType authentication)
+        [TestCase("ServicePrincipal")]
+        [TestCase("ServicePrincipalInKeyVault")]
+        public async Task CreateAndGetAzureDataExplorerDataFeedWithCredentialAuthentication(string authenticationType)
         {
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
+            var authentication = GetAuthenticationInstance<AzureDataExplorerDataFeedSource.AuthenticationType>(authenticationType);
 
             var credentialName = Recording.GenerateAlphaNumericId("credential");
-            await using var disposableCredential = await DisposableDataSourceCredentialEntity.CreateDataSourceCredentialEntityAsync(adminClient, credentialName, authentication.ToString());
+            await using var disposableCredential = await DisposableDataSourceCredentialEntity.CreateDataSourceCredentialEntityAsync(adminClient, credentialName, authenticationType);
```

Comment:
`authentication.ToString()` returns the inner string used by the service now. What we want now is what we call `authenticationType`.